.github/workflows/conformance-ingress.yaml

```diff
@@ -417,6 +417,17 @@ jobs:
             -enable-http-debug \
             -stop-on-failure
 
+      - name: Run basic CLI tests
```

Comment:
Step name and junit-property github_job_step needs to be unique and same
Since matrix.name is unique we can use that here.
`-name: Run basic CLI tests (${{ .matrix.name }})`